atd-vze/src/views/Crashes/Crash.js

```diff
@@ -37,23 +37,6 @@ import {
   DELETE_NOTE,
 } from "../../queries/crashNotes";
 
-const calculateYearsLifeLost = people => {
-  // Assume 75 year life expectancy,
-  // Find the difference between person.prsn_age & 75
-  // Sum over the list of ppl with .reduce
-  return people.reduce((accumulator, person) => {
-    let years = 0;
-    if (person.injury_severity.injry_sev_desc === "KILLED") {
-      let yearsLifeLost = 75 - Number(person.prsn_age);
-      // What if the person is older than 75?
-      // For now, so we don't have negative numbers,
-      // Assume years of life lost is 0
-      years = yearsLifeLost < 0 ? 0 : yearsLifeLost;
```

Comment:
awesome to see this gone and replaced with using the generated column from the db :broom: